src/components/FilterPanel.test.tsx

```diff
@@ -0,0 +1,67 @@
+import React, { Suspense } from 'react';
+import { render, screen, fireEvent } from '@testing-library/react';
+import { BrowserRouter } from 'react-router-dom';
+import FilterPanel from './FilterPanel';
+import { I18nextProvider } from 'react-i18next';
+import i18n from '../i18n';
+import { act } from 'react-dom/test-utils';
+
+afterAll(() => {
+  jest.clearAllMocks();
+});
+
+function renderComponent() {
+  return render(
+    <Suspense fallback="loading">
+      <I18nextProvider i18n={i18n}>
+        <BrowserRouter>
+          <FilterPanel onClose={() => {}} />
+        </BrowserRouter>
+      </I18nextProvider>
+    </Suspense>
+  );
+}
+
+test('renders component', async () => {
+  renderComponent();
+  expect(await screen.findByText('Filters')).toBeInTheDocument();
+  expect(screen.getByText('What are you studying?')).toBeInTheDocument();
+  expect(screen.getByText('Min Amount')).toBeInTheDocument();
+  expect(screen.getByText('Grade Level')).toBeInTheDocument();
+  expect(screen.getByText('State')).toBeInTheDocument();
+
+  expect(
+    screen.getByText('Your filters are currently applied.')
+  ).toBeInTheDocument();
+  expect(screen.getByText('Apply')).toBeInTheDocument();
+  expect(screen.getByText('Cancel')).toBeInTheDocument();
```

Comment:
FYI since these are all repetitive for future reference you could write these tests like this:
```jsx
[
  'What are you studying?', 
  ...
].forEach((s) => expect(screen.getByText(s)).toBeInTheDocument());
```

The array could be stored in a `const` too.

Reply:
Nice!